fineract-provider/src/main/java/org/apache/fineract/infrastructure/bulkimport/importhandler/savings/SavingsImportHandler.java

```diff
@@ -147,7 +148,9 @@ private SavingsAccountData readSavings(final Workbook workbook, final Row row, f
         Long interestPostingPeriodTypeId = null;
         EnumOptionData interestPostingPeriodTypeEnum = null;
         if (interestPostingPeriodType != null) {
-            if (interestPostingPeriodType.equalsIgnoreCase(MONTHLY)) {
+            if (interestPostingPeriodType.equalsIgnoreCase(DAILY)) {
```

Comment:
This seems out of context. Would you mind to create a separate PR for this? It has not much to do with 4 eye principle and maker-checker handling